.github/workflows/ci-sharktank-and-shortfin.yml

```diff
@@ -70,31 +70,5 @@ jobs:
             iree-runtime \
             "numpy<2.0"
 
-      - name: Run llama test
-        run: ./build_tools/integration_tests/llama_export_compile_serve.sh
-
-  test_punet:
-    name: "Integration Tests - punet"
-    runs-on: nodai-amdgpu-mi250-x86-64
-    env:
-      VENV_DIR: ${{ github.workspace }}/.venv
-    steps:
-      - name: "Checkout Code"
-        uses: actions/checkout@v3
-
-      - name: "Setup Python venv"
-        run: python3 -m venv ${VENV_DIR}
-
-      - name: Install pip deps
-        run: |
-          source ${VENV_DIR}/bin/activate
-          python -m pip install --no-compile --upgrade pip
-          pip install --no-compile -r pytorch-rocm-requirements.txt
-          pip install --no-compile -f https://iree.dev/pip-release-links.html --src deps \
-            -e "git+https://github.com/iree-org/iree-turbine.git#egg=iree-turbine"
-          pip install --no-compile -r requirements.txt -r sharktank/requirements-tests.txt -e sharktank/ shortfin/
-
-      - name: Run punet tests
-        run: |
-          source ${VENV_DIR}/bin/activate
-          pytest -v sharktank/ -m model_punet
+      - name: Run shortfin LLM Server Integration Test
+        run: pytest -v build_tools/integration_tests/
```

Comment:
Could keep this name generic, for when other tests are added to this folder.

```suggestion
      - name: Run integration tests
        run: pytest -v build_tools/integration_tests/
```

Or split into subfolders per component... but we can solve that when N > 1.

```yml
      - name: Run LLM integration tests
        run: pytest -v build_tools/integration_tests/llm

      - name: Run SD integration tests
        run: pytest -v build_tools/integration_tests/sd
```

Reply:
I went with the latter. Really like that setup. Will make it much easier to track where the error is and whether or not it has to do with your PR (i.e. I make an SD change unrelated to LLM)